Makefile

```diff
@@ -288,6 +288,17 @@ static-check: gocyclo
 
 get-deps: .get-deps-stamp
 
+DEP_VERSION=v0.4.1
+get-dep-linux:
```

Comment:
for my understanding really, but could this and -osx targets be added under get-deps-stamp? also, should these be marked phony?

Reply:
> added under get-deps-stamp?

i kept it separate since we want to vendor most of the `get-deps` items as dependencies using the dep tool. whereas the dep tool falls outside of that scope. 

> should these be marked phony?

good question. im not sure. 🤔

Reply:
As written, yes they should be.